NCrontab.Tests/CrontabScheduleTests.cs

```diff
@@ -54,12 +54,18 @@ public void CannotParseEmptyString()
         }
 
         [Test]
-        public void TryParseNullString() =>
+        public void TryParseNullString()
+		{
             Assert.That(CrontabSchedule.TryParse(null!), Is.Null);
+            Assert.False(CrontabSchedule.TryParse(string.Empty, out var _));
```

Comment:
A few things:

- The first argument to `TryParse` here should be `null`, not `string.Empty`.
- For consistency with the rest of the tests, use `Assert.That`.
- Testing that `result` is indeed `null` would be more thorough.

```suggestion
            Assert.That(CrontabSchedule.TryParse(null!, out var result), Is.False);
            Assert.That(result, Is.Null);
```